### ISSUE TITLE
Hidden table at buttom

### ISSUE THREAD

**Issue by torgeirl** [devilry/devilry-django#1203]
Below the footer, there seems to be a meta class `div` element containing a hidden table:
```
  ...
</footer>
<div class="cfp-hotkeys-container fade ng-scope" ng-class="{in: helpVisible}" style="display: none;"> == $0
  <div class="cfp-hotkeys">
  ...
  <table>
    <tbody>
      ...
```

This could cause issues with screen readers, and should be avoided.

**Comment by espenak** [comment]
This is not easy to get rid of without the previously discussed project to replace angularjs @torgeirl. We could easily make this invisible with some hacks, but that would break the places where this is used for hotkeys support.

**Comment by torgeirl** [comment]
@espenak: OK, removing this from the current milestone awaiting an Angular replacement/upgrade.
